New history tabs (#7594)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/account-settings.md

Once enabled, the **dbt compare** option becomes available in the CI job settings for you to select.

<Lightbox src="/img/docs/deploy/example-account-settings.png" width="85%" title="Example of the Enable account access to Advanced CI option" />

## Project settings history

You can view historical project settings changes over the last 90 days.

To view the change history: 

> 1. Navigate to **Account settings** from the main menu and click **Projects**. 

Reviewer: ```suggestion
1. Click your account name at the bottom of the left-side menu and click **Account settings**.
2. Click **Projects**. 
```

> 2. Click a **project name**.

Reviewer: ```suggestion
3. Click a **project name**.
```

> 3. Click **Settings**. 

Reviewer: I tried the steps and I didn't have to click Settings -- the Settings tab was automatically opened when I clicked the project. 
```suggestion
```

Author: Good call out! This step is for a different approach
